certbot/plugins/pluginstorage_test.py

```diff
@@ -0,0 +1,140 @@
+"""Tests for certbot.plugins.common.PluginStorage"""
+import json
+import mock
+import os
+import shutil
+import tempfile
+import unittest
+
+from certbot import errors
+
+from certbot.plugins import common
+
+
+class PluginStorageTest(unittest.TestCase):
```

Comment:
I think this class should inherit from `certbot.tests.util.ConfigTestCase` and inherit the config and automatic teardown. You'd just want to create the `self.config.config_dir` in `setUp` here and the `tearDown` in `ConfigTestCase` will clean it up.